src/NLog/Targets/DatabaseTarget.cs

```diff
@@ -392,34 +392,63 @@ protected override void InitializeTarget()
                 }
                 catch (Exception ex)
                 {
-                    InternalLogger.Warn(ex, "DatabaseTarget(Name={0}): DbConnectionStringBuilder failed to parse ConnectionString", Name);
+#if !NETSTANDARD
+                    if (!string.IsNullOrEmpty(ConnectionStringName))
+                        InternalLogger.Warn(ex, "DatabaseTarget(Name={0}): DbConnectionStringBuilder failed to parse ConnectionString: {1}", Name, ConnectionStringName);
```

Comment:
I would expect the connectionstring after `ConnectionString:`.  Maybe change it to `ConnectionString name`?